Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderThumbnailView.cpp

```diff
@@ -814,11 +815,12 @@ namespace AzQtComponents
                                                            : QItemSelectionModel::SelectionFlag::Select);
                     for (auto index : indexRange.indexes())
                     {
-                        selectionModel()->select(index, selectionFlag);
+                        (isRightClick ? selectionModel()->select(index, QItemSelectionModel::SelectionFlag::Select)
+                                      : selectionModel()->select(index, selectionFlag));
```

Comment:
Same comment as above about expanding or reformatting this condition

Reply:
Done!